spark/README.md

```diff
@@ -22,18 +22,18 @@ version to build.
 ```bash
 git clone https://github.com/apache/spark
 cd spark
-git checkout v3.1.1
+git checkout branch-3.4
```

Comment:
The tag for `v3.4.0` or `v3.5.0` exists. Is this one a remote branch or?

Reply:
Yeah, this is the branch name from the official repo. I guess the naming convention changed
https://github.com/apache/spark/tree/branch-3.4

Reply:
Branches might disappear. And you will get unstable patches if you point to the branch path. It's safer to use the tags on the master for each version (like `v3.4.1`)